pkg/apis/agones/v1/gameserver.go

```diff
@@ -700,6 +700,20 @@ func (gs *GameServer) IsBeforeReady() bool {
 	return false
 }
 
+// IsReady returns true if the GameServer status is Ready, Reserved, or Allocated state.
+func (gs *GameServer) IsReady() bool {
```

Comment:
Late to the party:

Maybe `IsActive` ? Since `Ready` is the Ready state, this might get confusing?

Not sure how much I like the idea of `Active` either, but I figure `Ready` is more confusing. WDYT?

Reply:
I like it. Since this is already merged I'll make the change in the next PR.